src/Equinor.ProCoSys.IPO.Command/InvitationCommands/InvitationHelper.cs

```diff
@@ -80,6 +81,27 @@ public static string GenerateMeetingDescription(Invitation invitation, string ba
             meetingDescription += $"</br><a href='{baseUrl}" + $"/InvitationForPunchOut/{invitation.Id}'>" + "Open invitation for punch-out in ProCoSys.</a>";
             meetingDescription += $"</br><p>Best regards, {organizer.FirstName} {organizer.LastName}</p>";
 
+            if (isFake)
+            {
+                meetingDescription += string.Format(
```

Comment:
simplyfy with string interpolation. Argument 2 and 3 not needed at all since you insert blank